src/System.Private.CoreLib/shared/System/Threading/ThreadPool.cs

```diff
@@ -189,7 +189,7 @@ public void LocalPush(object obj)
                         if (count >= m_mask)
                         {
                             // We're full; expand the queue by doubling its size.
-                            var newArray = new object[m_array.Length << 1];
+                            var newArray = new object?[m_array.Length << 1];
```

Comment:
Is this API missing a throw if obj is null?  It doesn't seem like obj can be null.

https://github.com/dotnet/coreclr/pull/23600/files#diff-5eb107b5c9cc6b65a845d39ee38e7f13R135

Reply:
Same question for `LocalFindAndPop`, `Enqueue` and others.

Reply:
These are on internal types.  Do your questions still apply?

Reply:
Ah I see. I saw that the signature was public, missed that the types where internal... Since the array can have null values in it, why can't LocalPush accept nullables then?

Reply:
The only thing being pushed are non-null callback objects:
https://github.com/dotnet/coreclr/blob/ef0fde985732148eea136e90aece4ccd3d0370b5/src/System.Private.CoreLib/shared/System/Threading/ThreadPool.cs#L466
Allowing the argument to be null wouldn't add any benefit.

Reply:
Ah ok, cool. Thanks for clarifying.